AdminUi/apps/admin_ui/lib/core/modals/add_quota_dialog.dart

```diff
@@ -92,32 +92,32 @@ class _AddQuotaDialogState extends State<_AddQuotaDialog> {
                 value: _selectedMetric,
                 items: widget.availableMetrics.map((metric) => DropdownMenuItem(value: metric.key, child: Text(metric.displayName))).toList(),
                 onChanged: _saving ? null : (String? selected) => setState(() => _selectedMetric = selected),
-                decoration: const InputDecoration(
-                  border: OutlineInputBorder(),
-                  labelText: 'Metric*',
+                decoration: InputDecoration(
+                  border: const OutlineInputBorder(),
+                  labelText: context.l10n.metric,
```

Comment:
In the MeinBildungsraum app we don't want to have the `*` in the translation itself. We do: `'${context.l10n.metric}*'`

ping @jkoenig134: correct?

Reply:
correct, please do that.

Reply:
Changes made.